src/Grb.Building.Api/appsettings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "ConnectionStrings": {
+    "BuildingGrb": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.BuildingRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "BuildingGrbAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.BuildingRegistry;Trusted_Connection=True;TrustServerCertificate=True;"
+  },
+
+  "OAuth2IntrospectionOptions": {
+    "ClientId": "acmidmconsumer.webapi",
+    "ClientSecret": "a_very=Secr3t*Key",
+    "Authority": "http://localhost:5051",
+    "IntrospectionEndpoint": "http://localhost:5051/connect/introspect"
+  },
+
+  "NisCodeServiceUrl": "https://niscode.ops-api.basisregisters.dev-vlaanderen.be",
+
+  "Bucket": {
+    "BucketName": "basisregisters-dev-building-grb-uploads",
+    "UrlExpirationInMinutes": 60
+  },
+
+  "AWS": {
+    "Region": "eu-west-1"
+  },
+
+  "DataDog": {
+    "Enabled": true,
+    "Debug": false,
+    "ServiceName": "building-registry-grb-upload-api"
```

Comment:
@ArneD Do we need to change this to basisregisters-grb-upload-api?

Reply:
no, it's all good